### DISCUSSION TITLE
Plex Meta Manager collections not updating

### DISCUSSION THREAD

**Discussion by groaninjock** [meisnate12/Plex-Meta-Manager#789] [Q&A]
I've spent the last few days setting up Plex Meta Manager, and thought I had it running as I wanted it to - smart collections for specific directors, which I thought would auto-update every night if I added new content by those directors.

Yesterday I added new content by a couple of those directors - Adam McKay and Richard Linklater. Checking Plex this morning, the collections for both of those directors were empty, with even the 'old' movies missing. If I cleared the filter in the smart collection and reapplied it (in Plex app), then all of the movies appeared as they should, including the new ones.

I'm guessing that I've got a setting wrong somewhere, but I don't really know where to start with troubleshooting it. Does anyone know where I might be going wrong?

**Comment by kevbentz** [comment]
Do you have it configured to delete unmanaged collections?
